omod/src/main/webapp/pages/home.gsp

```diff
@@ -26,7 +26,7 @@
     </div>
 </div>
 <div class="row">
-    <div  class="col-12 col-sm-12 col-md-12 col-lg-12 homeList">
+    <div  class="col-12 col-sm-12 col-md-12 col-lg-12 homeList" id="apps">
```

Comment:
@Akayeshmantha Where is the id for the div used?

Reply:
Hi @ssmusoke  it is used in the selenium test. Btw it's weird this change was already merged to the branch.I think issue on my fork it seems will remove it and push again with a new PR